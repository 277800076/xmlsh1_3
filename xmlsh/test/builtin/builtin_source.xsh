# Test of source (.) command

if [ -z "$_SOURCE" ] ; then
	echo top level invocation of $0
	_SOURCE=source
	. $0 
	exit 0
else
	echo sourced invocation of $0
	echo _SOURCE is _SOURCE
fi
unset _SOURCE
