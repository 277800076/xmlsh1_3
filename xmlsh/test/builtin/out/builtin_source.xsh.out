top level invocation of builtin_source.xsh
sourced invocation of builtin_source.xsh
_SOURCE is _SOURCE
